packages/keycloak-user-management/src/components/forms/UserForm/utils.tsx

```diff
@@ -228,3 +228,23 @@ export const getUserFormPayload = (values: FormFields) => {
     userGroups: values.userGroups ?? [],
   };
 };
+
+/**
+ * converts userGroups to options that can be passed to antd Select
+ *
+ * @param userGroups - the userGroups in an array
+ */
+export const getUserGroupsOptions = (userGroups: UserGroup[]): SelectOption[] =>
+  userGroups.map((userGroup) => {
+    return { value: userGroup.id, label: userGroup.name };
```

Comment:
should these have a key?

Reply:
I do not think so, at least the docs do not say it should be added, the docs-defined interface is `{ label, value }[]`